inc/userinjection.class.php

```diff
@@ -185,6 +185,33 @@ public function processAfterInsertOrUpdate($values, $add = true, $rights = [])
             /** @phpstan-ignore-next-line */
             $DB->query($query); // phpcs:ignore
         }
+
+        if (isset($values['User']['profiles_id'])) {
+            $profile_user = new Profile_User();
+            $profile_user->getFromDBByCrit([
+                'users_id' => $values['User']['id'],
+                'entities_id' => $values['User']['entities_id'],
+                'prodiles_id' => $values['User']['profiles_id']
```

Comment:
I doubt it worked with the typo:
```suggestion
                'profiles_id' => $values['User']['profiles_id']
```
Can you retest the scenario described by @Lainow ?